.idea/store_inventory.iml

```diff
@@ -7,9 +7,11 @@
     <orderEntry type="module-library">
```

Comment:
FYI this needs to not be in git, since it's what phpstorm uses to keep track of things. Add it to .gitignore on new projects